### ISSUE TITLE
 error   options must be an object

### ISSUE THREAD

**Issue by mino89** [storyblok/storyblok-astro#53]
I am trying to implement storyblok in a fresh new astro project, but I continue to receive this error from axios

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/5948240/199621270-598da0d4-39ee-4c5e-a984-644a8f9b4918.png">

here is my astro.config
```js

import { defineConfig } from 'astro/config';
import storyblok from "@storyblok/astro";

export default defineConfig({
  integrations: [
    storyblok({
      accessToken: "***",
      apiOptions: {
        cache: { clear: "auto", type: "memory" },
      },
      bridge: true,
      components: {
        page: "storyblok/Page",
        feature: "storyblok/Feature",
        grid: "storyblok/Grid",
        teaser: "storyblok/Teaser",
      },
    }),
  ]
});
```
here is the page that generates the error
```js
---
import { useStoryblokApi } from "@storyblok/astro";
import StoryblokComponent from "@storyblok/astro/StoryblokComponent.astro";
import BaseLayout from "../layouts/BaseLayout.astro";

const storyblokApi = useStoryblokApi();
const { data } : any = await storyblokApi.get("cdn/stories/landing", {
  version: "draft"
});

const story = data.story;

const pageTitle = "Landing";

---
<BaseLayout pageTitle={pageTitle}>
<StoryblokComponent blok={story.content} />
</BaseLayout>
```
[here the link](https://github.com/mino89/astro-blog-tutorial) of my repo

**Comment by nintra** [comment]
Had the same problem. Something is odd with the apiPlugin. 
It works if you use `client` for making API requests.
```
const { client, accessToken } = useStoryblokApi()
const result = await client.get('cdn/stories/landing', {
    params: {
        version: 'draft',
        token: accessToken,
    },
})
```

**Comment by mino89** [comment]
Thank you @nintra !
I'll make a pull request for the update of the documentation

**Comment by manuelschroederdev** [comment]
That's a weird one. Which `axios` version do you have installed, @mino89? Thankfully, all of our SDKs, including this one for Astro, will soon get rid of `axios` entirely.

**Comment by philr35** [comment]
Hey y'all, I have the same error and figured out the root cause.


```
error   options must be an object
  File:
    /Users/phillip.rognerud/Desktop/tpw/frontend/node_modules/axios/dist/node/axios.cjs:3435:11
  Code:
      3434 |   if (typeof options !== 'object') {
    > 3435 |     throw new AxiosError('options must be an object', AxiosError.ERR_BAD_OPTION_VALUE);
           |           ^
      3436 |   }
      3437 |   const keys = Object.keys(options);
      3438 |   let i = keys.length;
    
  Stacktrace:
AxiosError: options must be an object
```

The issue is with `const storyblokApi = useStoryblokApi();` and with [Axios version](https://github.com/axios/axios/releases) `>=1.2.0`

**To fix the issue, downgrade axios to version `1.1.0`**
